ui/lib/config-ui/addon/components/messages/page/create-and-edit-message-form.hbs

```diff
@@ -67,12 +67,21 @@
         data-test-button="preview"
       />
 
-      <Hds::Button
-        @text="{{if @message.isNew 'Create' 'Edit'}} message"
-        disabled={{(or this.invalidFormMessage this.save.isRunning)}}
-        data-test-button="create-message"
-        type="submit"
-      />
+      {{#if this.hasSomeActiveModals}}
+        <Hds::Button
+          @text="{{if @message.isNew 'Create' 'Edit'}} message"
+          disabled={{(or this.invalidFormMessage this.save.isRunning)}}
```

Comment:
A while ago we stopped the pattern of disabling submit buttons if form validations failed, so we can remove this first conditional!